app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -43,7 +43,7 @@ def host_products(host)
       def host_products_names(host)
         products = host_products(host)
         if products
-          products.map(&:name).join(', ')
+          products.map(&:name)
         else
           ''
```

Comment:
should this be `[]` instead?

Reply:
Likely the else can be removed entirely

Reply:
i think the condition is needed because `host_products` can return nil

Reply:
Oh :confounded: